website/docs/patterns/form-patterns/partials/validation/validation.md

```diff
@@ -1,40 +1,37 @@
-!!! Warning
-
-While we provide the structure and visual consistency for validation, the messaging and functionality need to be implemented by the product teams.
-!!!
-
-Form validation ensures that fields are correctly formatted, required fields are not empty, and the data being submitted matches the requirements for application. A successful validation strategy relies on three key concepts:
+Form validation ensures that fields are correctly formatted, required fields are not empty, and the data being submitted matches the requirements for the application. A successful validation strategy relies on three key concepts:
 
-1. **Uses clear language:** the validation message should be human readable and easy to understand.
-2. **Displayed at the right time:** The validation message should be displayed as soon as possible to when the error occurred.
-3. **Displayed in the right place:** the validation message should be noticeable and located where the error occurred.
+1. **Clear language:** The validation message should be human-readable and easy to understand.
+2. **Placement:** The validation message should be displayed as soon as possible after the error occurred.
+3. **Timing:** The validation message should be noticeable and located where the error occurred.
 
-!!! Info
+!!! Warning
 
-At this time Helios components only support error validation. If you have a need for success validation in your application please submit a [request](/about/support).
+While we provide the structure and visual consistency for validation, the messaging and functionality need to be implemented by the product teams.
 !!!
 
-## Types of validation
+## Clear validation language
 
-We recommend using a combination of client-side and server-side validation for the best user experience.
+Error messages should clearly, but briefly communicate the issue for each impacted input. Using plain language to state the problem and suggested solution helps reduce friction when users are resolving errors in their inputs. 
 
-### Client-side
+![Example of credit card information form with invalid expiration date](/assets/patterns/form-patterns/form-pattern-validation-error-message-example.png)
```

Comment:
There's extra whitespace included at the bottom of the example image causing extra space between the image and the following text. Could this be trimmed?

<img width="894" alt="image" src="https://github.com/user-attachments/assets/c95e8498-0121-44e8-b4eb-31625e971c07" />